### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.4.1 - make navigation elements distinguishable for screen reader users

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1291]
# Feature Request

## Description
If a page has more than one nav-Elements, each nav must be distinguishable to screen reader users. 

## Solution
add aria-labels to `<nav>` elements: 
- "main navigation"
- "subnavigation"

Note: The breadcrumb navigation already has an aria-label.

**Comment by alboth** [comment]
@alexakressmann @SarahEdding - which aria-label should be applied. pls define. thx

**Comment by SarahEdding** [comment]
correct, aria-labels must be added to `<nav>`:
`<nav id="mainnavigation" class="collapse navbar-collapse" aria-label="main navigation">` 
and 
`<nav aria-label="subnavigation" class="frame frame-type-subnavigation frame-background-none frame-no-backgroundimage frame-space-before-none frame-space-after-none" role="navigation">`

for subnavigation see https://bitv-v12.live.typo3.cps.321.works/content-examples/menus/pages 
![image](https://github.com/benjaminkott/bootstrap_package/assets/132338236/262d582e-c3b2-4dcf-bb88-ad9cd7c9124d)

**Comment by alboth** [comment]
@alboth finished this ticket. @SarahEdding thank you for your information. 
merge request was sent

**Comment by SarahEdding** [comment]
QA done, all fixed :)
